hedera-mirror-importer/src/main/resources/db/migration/v1/V1.50.3__account_alias.sql

```diff
@@ -0,0 +1,16 @@
+-------------------
+-- Add support for automatic account creation alias
+-------------------
+
+-- Add nullable alias column
+alter table if exists entity
+    add column if not exists alias bytea null;
+
+alter table if exists entity_history
+    add column if not exists alias bytea null;
+
+-- support retrieval by alias
+create index if not exists entity__alias_deleted
+    on entity (alias, deleted);
+create unique index if not exists entity__alias_id
+    on entity (alias, id);
```

Comment:
I don't think we need two alias indexes. Alias is already distinct except for the reuse case and reuse is not going to occur enough to warrant adding delete to it. I don't think we need a unique index with id either as id will not be in any queries that use this index.

```suggestion
create index if not exists entity__alias
    on entity (alias);
```

Reply:
Yeah with the simplified flow it's not needed. Removed